gengokit/generator/gen_test.go

```diff
@@ -116,7 +115,7 @@ func svcMethodsNames(methods []*svcdef.ServiceMethod) []string {
 	return mNames
 }
 
-func stringToTemplateExector(def, importPath string) (*gengokit.TemplateExecutor, error) {
+func stringToTemplateExector(def, importPath string) (*gengokit.Executor, error) {
```

Comment:
`Data` works. I generally have a distaste for this as an identifier because of its extreme overuse in some of the legacy code here, all over the place, where its use does not communicate any meaning at all. As long as its use is restricted to narrowly-scoped cases, it's fine.